apps/web/components/imports/destination/Destination.tsx

```diff
@@ -5,7 +5,7 @@ import { Stack, TextInput as Input, Group } from '@mantine/core';
 import { Button } from '@ui/button';
 import { NumberInput } from '@ui/number-input';
 import { DoaminInput } from '@ui/domain-input';
-import { REGULAR_EXPRESSIONS } from '@config';
+import { documentationReferenceLinks, REGULAR_EXPRESSIONS } from '@config';
```

Comment:
@thisismayuresh Rename `documentationReferenceLinks` to `DOCUMENTATION_REFERENCE_LINKS`